feature-libs/asm/jest.ts

```diff
@@ -22,3 +22,15 @@ Object.defineProperty(document.body.style, 'transform', {
   },
 });
 */
+
+import 'zone.js/fesm2015/zone-testing-bundle.min.js';
```

Comment:
1. Do we need to include zone also for unit tests of schematics? I guess no.
2. Btw. why not just `import 'zone.js'`?

Reply:
Simple `import 'zone.js'` breaks unit tests. I assume this specific version is prepared to be working with NG13 somehow, but I didn't dive into the internals.

Reply:
1. Did you try importing both zone.js and zone.js/testing, just as we to in files `test.ts`, e.g.:
https://github.com/SAP/spartacus/blob/ebcb4c9e582d385ab817bb384a07abb6715ece48/projects/core/src/test.ts#L3-L5

2. Could you attach here the error message when using those imports, please?

Reply:
Actually importing both lines works well. I get green and finished results. I assume we want to go this way? (2 imports as mentioned above)

Reply:
If you confirm it helps, I'm for it.